Automattic/newspack-plugin
# feat: reorganize segment UI

### All Submissions:






### Changes proposed in this Pull Request:



Improvements to segment editing UI.

Originally sourced from https://github.com/Automattic/newspack-plugin/pull/848.

Closes #828.

### How to test the changes in this Pull Request:

1. In Campaigns wizard, visit the Segments tab and click on a segment's name to edit it
2. Observe a new UI in which the segmentation criteria are compartmentalised to improve readability
3. Observe a notice in the "Referrer sources" informing about this criterion being applied to first page view only

### Other information:
